### PR TITLE
Improvement / HOLO 772 indexer processor handle mint integration

### PR DESCRIPTION
**Pull request by BravoNatalie**
## Describe Changes

- add sqs handler for the mint event and sqs integration

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] I have checked eslint

### CODE REVIEW

File: src/handlers/indexer/handle-mint-event.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import {networkToChainId} from '../../utils/utils'
+import {PayloadType, SqsMessageBody} from '../../types/sqs'
+
+async function handleMintEvent(
+  networkMonitor: NetworkMonitor,
+  transaction: TransactionResponse,
+  network: string,
+  tags: (string | number)[],
+): Promise<void> {
+  const receipt: TransactionReceipt | null = await networkMonitor.getTransactionReceipt({
```
Comment (sogoiii) [inline src/handlers/indexer/handle-mint-event.ts]: @BravoNatalie @alexanderattar 

So i think we can push more work to job consumer. The CLI only has to detect the new event and send a job. That way we simplify the CLI to be a block / event parser.

File: src/commands/indexer/index.ts [hunk 2]
```diff
@@ -395,14 +395,7 @@ export default class Indexer extends HealthCheck {
...
               )
-              await handleMintEvent.call(
-                this,
-                this.networkMonitor,
-                transaction,
-                job.network,
-                tags,
-                this.updateMintedERC721,
-              )
+              await handleMintEvent.call(this, this.networkMonitor, transaction, job.network, tags)
```
Comment (sogoiii) [inline src/commands/indexer/index.ts]: @BravoNatalie @alexanderattar I also think we should have the original handler and the new handler that submits a job. That way we can transition from on version to anther. Aka `handlers/handle-mint-events.ts` is the original and the new `handler/handle-mint-eventsv2.ts` that only submits a job. Or we just make a new generic job sender function that is called here. In short, what would be a good method to follow to transition into the new setup.

**Review by sogoiii — CHANGES_REQUESTED** [review]

File: src/commands/indexer/index.ts (hunk 2 above)
Comment (BravoNatalie) [reply src/commands/indexer/index.ts]: I agree, we can have both implementations for the transition.

**Review by sogoiii — APPROVED** [review]
👍

**Pull request merged**